src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Logging/ProjectOutputWindowProjectLogger.cs

```diff
@@ -41,17 +41,18 @@ public void WriteLine(StringFormat format)
                 string text = format.Text + Environment.NewLine;
 
                 // Extremely naive implementation of a Windows Pane logger - the assumption here is that text is rarely written,
-                // so transitions to the UI thread are uncommon and are fire and forget. If we start writing to this a lot (such 
+                // so transitions to the UI thread are uncommon and are fire and forget. If we start writing to this a lot (such
                 // as via build), then we'll need to implement a better queueing mechanism.
                 _threadingService.Fork(async () =>
                 {
 
                     IVsOutputWindowPane pane = await _outputWindowProvider.GetOutputWindowPaneAsync()
                                                                           .ConfigureAwait(true);
-                                                                          
+
                     pane.OutputStringNoPump(text);
 
-                }, options: ForkOptions.HideLocks | ForkOptions.StartOnMainThread);
+                }, options: ForkOptions.HideLocks | ForkOptions.StartOnMainThread,
+                   factory: _threadingService.JoinableTaskFactory);
```

Comment:
FYI I filed https://github.com/Microsoft/vs-threading/issues/139 for this, since it wasn't strictly necessary. But it's harmless to do so for the sake of suppressing just this instance of a diagnostic without suppressing the whole rule, I support this change.